packages/react/src/date-picker/Calendar.tsx

```diff
@@ -297,7 +291,6 @@ const calendarComponents: CustomComponents = {
 				as="h2"
 				fontSize="lg"
 				fontWeight="bold"
-				id={id}
 				lineHeight="heading"
 			>
 				{formattedMonthYear}
```

Comment:
Interesting. This heading isn't correct for date range, because it can show 2 months. The new `aria-label` on the tables are correct